contracts/contracts/NoLossDao.sol

```diff
@@ -107,7 +107,7 @@ contract NoLossDao is Initializable {
   }
 
   modifier proposalActive(uint256 propId) {
-    require(state[propId] == ProposalState.Active, "Proposal isn't active");
+    require(state[propId] == ProposalState.Active, 'Proposal is not active');
```

Comment:
We must commit a sollint file config so that it does'nt keep changing "->' etc...